cli/command/container/logs.go

```diff
@@ -50,6 +50,11 @@ func NewLogsCommand(dockerCli command.Cli) *cobra.Command {
 func runLogs(dockerCli command.Cli, opts *logsOptions) error {
 	ctx := context.Background()
 
+	c, err := dockerCli.Client().ContainerInspect(ctx, opts.container)
```

Comment:
It would probably make sense to have the call to container logs use `c.ID` instead of the passed in name.